beta/src/components/MDX/Sandpack/CustomPreset.tsx

```diff
@@ -8,16 +8,19 @@ import {flushSync} from 'react-dom';
 import {
   useSandpack,
   useActiveCode,
-  SandpackCodeEditor,
   SandpackThemeProvider,
 } from '@codesandbox/sandpack-react';
+import { SandpackCodeEditor } from "@codesandbox/sandpack-react/dist/components/CodeEditor"
+
 import scrollIntoView from 'scroll-into-view-if-needed';
 
 import {IconChevron} from 'components/Icon/IconChevron';
 import {NavigationBar} from './NavigationBar';
 import {Preview} from './Preview';
 import {CustomTheme} from './Themes';
 
+// const SandpackCodeEditor = React.lazy(() => import('@codesandbox/sandpack-react/dist/components/CodeEditor').then(module => module.SandpackCodeEditor) as any)
```

Comment:
👀  Just curious, why have we used React.lazy instead of the dynamic import that is provided by Next.js?

https://nextjs.org/docs/advanced-features/dynamic-import